src/main/webapp/store.html

```diff
@@ -8,17 +8,21 @@
     <script src="map.js"></script>
   </head>
   <body onload="createMap(); determineLogin();" id="background">
-    <nav id="navbar">
-        <a href="index.html" class="button">Home</a>
-        <a href="user.html" class="button">Profile</a>
-        <a href="store.html" id= "active" class="button">Explore</a>
-        <a href="search.html" class="button">Search for a drink!</a> 
-    </nav>
+
+  <nav id="navbar">
+   <a href="index.html" id="active" class="button">Home</a>
+   <a href="user.html" class="button">Profile</a>
+   <a href="store.html" class="button">Explore</a>
```

Comment:
explore.html -> should be active in navbar, not index

Reply:
Didn't notice the mistake.